routes/index/sold-tracking.html

```diff
@@ -0,0 +1,8 @@
+<!-- Twitter single-event website tag code -->
+<!--
+<script src="//platform.twitter.com/oct.js" type="text/javascript"></script>
+<script type="text/javascript">twttr.conversion.trackPid('nxrhi', { tw_sale_amount: 0, tw_order_quantity: 0 });</script>
+-->
+<img height="1" width="1" style="display:none;" alt="" src="https://analytics.twitter.com/i/adsct?txn_id=nxrhi&p_id=Twitter&tw_sale_amount=0&tw_order_quantity=0" />
+<img height="1" width="1" style="display:none;" alt="" src="//t.co/i/adsct?txn_id=nxrhi&p_id=Twitter&tw_sale_amount=0&tw_order_quantity=0" />
```

Comment:
Just double checking -- Twitter includes two pixels in its embed? The domain differs for these images but otherwise they appear the same. My concern would be that two pixels (if the same) could cause double tracking. I don't have access to the ads backend atm to generate a tag for comparison, but wanted to confirm.